eo-maven-plugin/src/main/java/org/eolang/maven/ParseMojo.java

```diff
@@ -99,6 +99,15 @@ public final class ParseMojo extends SafeMojo {
         defaultValue = "true")
     private boolean failOnError = true;
 
+    /**
+     * Default threads number.
```

Comment:
@holydrug This parameter is not for _default threads number_ but rather for just number of parallel threads.